gatekeeper.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: gatekeeper
+  version: 3.12.0
+  epoch: 0
+  description: Gatekeeper - Policy Controller for Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - go
+      - ca-certificates-bundle
+
+pipeline:
+  # We can't use go/install because this requires specific ldflags to set the version
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-policy-agent/gatekeeper
+      tag: v${{package.version}}
+      expected-commit: 770d9756db0487d1baa4079476c853236961a4eb
+      destination: gatekeeper
+
+  - runs: |
+      cd gatekeeper
+      FRAMEWORKS_VERSION=$(go list -f '{{ .Version }}' -m github.com/open-policy-agent/frameworks/constraint)
+      OPA_VERSION=$(go list -f '{{ .Version }}' -m github.com/open-policy-agent/opa)
+      go build -mod vendor -a -ldflags "-X github.com/open-policy-agent/gatekeeper/pkg/version.Version=v${{package.version}} -X main.frameworksVersion=${FRAMEWORKS_VERSION} -X main.opaVersion=${OPA_VERSION}" -o manager
```

Comment:
Do we need this two env to be set ?
`GO111MODULE=on` [here](https://github.com/open-policy-agent/gatekeeper/blob/master/Makefile#LL245C2-L245C16
) 
And 
`CGO_ENABLED=0 `[here](https://github.com/open-policy-agent/gatekeeper/blob/master/Dockerfile#LL17C4-L17C18)